packages/rune-games-cli/src/flows/Upload/Upload.tsx

```diff
@@ -96,9 +113,21 @@ export function Upload({ flags }: { flags: CliFlags }) {
                 />
               )}
 
+              {!!gameId && !buildCheckComplete && buildRequired(gameDir) && (
```

Comment:
Maybe we should move this before selecting the game?

Reply:
Sorry for the delay, thats in now.